estate/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': 'Estate',
+    'version': '0.0',
+    'depends': ['base'],
+    'author': 'haall-odoo',
```

Comment:
Fine in this tutorial, just know that when developing an Odoo module, we must put `Odoo S.A.`.